src/test/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck has started work.");
```

Comment:
The message 'Truck has started work.' could be more informative. It should indicate that it is a Truck that is working. Consider rephrasing to 'The Truck is now working.' or similar.